packages/terra-arrange/src/terra-dev-site/doc/arrange/Arrange.doc.jsx

```diff
@@ -24,19 +24,15 @@ const DocPage = () => (
         title: 'Align All Containers',
         example: (
           <div>
-            <ArrangeAlignment alignment="all" fitStart={image} fill={simpleText} />
-            <ArrangeAlignment alignment="all" fill={simpleText} fitEnd={icon} />
-            <ArrangeAlignment alignment="all" fitStart={image} fill={simpleText} fitEnd={icon} />
+            <ArrangeAlignment alignment="all" fitStart={alignExampleDiv} fill={simpleText} fitEnd={alignExampleDiv} />
           </div>
         ),
       },
```

Comment:
want to use the `source` attribute to toggle the example code?

Reply:
I'm currently in the process of adding this. :) I'm attempting to restructure the examples so the source examples will make sense individually, rather than just being the monolithic one currently being used.

Reply:
This is implemented in the current deployment.